experimental/plugins/lando-platformsh/services/platformsh-varnish/builder.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+// Modules
+const _ = require('lodash');
+
+// Builder
+module.exports = {
+  name: 'platformsh-varnish',
+  config: {
+    confSrc: __dirname,
+    port: '8086',
+    supportedIgnore: true,
+  },
+  parent: '_platformsh_service',
+  builder: (parent, config) => class LandoPlatformshVarnish extends parent {
+    constructor(id, options = {}, factory) {
+      options = _.merge({}, config, options);
+
+      // Build varnish
+      const varnish = {
+        image: `docker.registry.platform.sh/varnish-${options.version}`,
+        ports: [options.port],
+        environment: {
+          LANDO_WEBROOT_USER: options.meUser,
+          LANDO_WEBROOT_GROUP: options.meUser,
+        },
+        networks: {
+          default: {
+            aliases: ['varnish'],
```

Comment:
@mikemilano what is the purpose of the alias usage here and in the other services in this PR? All the services should automatically get `NAME.internal` aliases plus the ones lando gives by default.

Reply:
I had copied the networks block into a few services as I stubbed them out. I removed them all.